src/Network/Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -107,7 +102,6 @@ public class SetAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
 
         [Parameter(
             Mandatory = false,
-            ValueFromPipelineByPropertyName = true,
```

Comment:
Missed one case where this change was not reverted.

Reply:
Fixed